x-pack/plugins/observability_solution/apm/public/components/app/service_groups/service_group_save/group_details.tsx

```diff
@@ -31,22 +31,29 @@ interface Props {
   onClickNext: (serviceGroup: StagedServiceGroup) => void;
   onDeleteGroup: () => void;
   isLoading: boolean;
+  titleId?: string;
 }
 
+const DEFAULT_COLOR = '#5094C4';
+
 export function GroupDetails({
   isEdit,
   serviceGroup,
   onCloseModal,
   onClickNext,
   onDeleteGroup,
   isLoading,
+  titleId,
 }: Props) {
-  const [name, setName] = useState<string>(serviceGroup?.groupName || '');
+  const [name, setName] = useState(serviceGroup?.groupName);
   const [color, setColor, colorPickerErrors] = useColorPickerState(
-    serviceGroup?.color || '#5094C4'
+    serviceGroup?.color || DEFAULT_COLOR
```

Comment:
Can you create a variable for this and use it on line 55 as well?

```
const defaultColor = serviceGroup?.color || INITIAL_COLOR
```
```suggestion
    defaultColor
```

Reply:
@cauemarcondes sure, will do